apps/frontend/src/libs/components/meeting-form/libs/meeting-form.default-values.ts

```diff
@@ -0,0 +1,8 @@
+import { type CreateMeetingRequestDto } from "~/libs/types/types.js";
+
+const MEETING_FORM_DEFAULT_VALUES: CreateMeetingRequestDto = {
```

Comment:
```suggestion
const CREATE_MEETING_FORM_DEFAULT_VALUES: CreateMeetingRequestDto = {
```
Don't forget to change the file name as well